src/Sylius/Bundle/ApiBundle/Helpers/AbstractDataPricesTransformer.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Sylius\Bundle\ApiBundle\Helpers;
+
+use Sylius\Bundle\MoneyBundle\Form\DataTransformer\SyliusMoneyTransformer;
+
+abstract class AbstractDataPricesTransformer
+{
+    protected bool $isOutput;
+    protected SyliusMoneyTransformer $transformer;
+
+    public function __construct()
+    {
+        $this->transformer = CustomMoneyTransformerInitializer::getPriceTransformer();
+    }
+
+    protected function convertPrices(array $elements, array $targets): array
+    {
+        array_walk($elements, function (&$e) use ($targets) {
+            if (in_array($e['type'], $targets)) {
+                $e['configuration'] = $this->transformPriceFromConfiguration($e['configuration']);
+            }
+        });
+
+        return $elements;
+    }
+
+    protected function transformPriceFromConfiguration($configuration): array
+    {
+        foreach ($configuration as $k => $v) {
+            if (isset($configuration[$k]['amount'])) {
+                $amount = $this->transformPriceValue($configuration[$k]['amount']);
+                $configuration[$k]['amount'] = $amount;
+            }
+
+            if ($min = $configuration[$k]['filters']['price_range_filter']['min'] ?? null) {
+                $amountMin = $this->transformPriceValue($min);
+                $configuration[$k]['filters']['price_range_filter']['min'] = $amountMin;
+            }
+
+            if ($max = $configuration[$k]['filters']['price_range_filter']['max'] ?? null) {
+                $amountMax = $this->transformPriceValue($max);
+                $configuration[$k]['filters']['price_range_filter']['max'] = $amountMax;
+            }
+        }
+
+        return $configuration;
+    }
+
+    /**
+     * @param $value
+     * @return int|string|null
+     * @throws \Exception
+     */
+    protected function transformPriceValue($value)
+    {
+        if (!isset($this->isOutput)){
+            throw new \Exception('isOutput property has not been initialized');
+        }
+        if (gettype($value) !== "string") {
+            $value = strval($value);
+        }
```

Comment:
```suggestion
        if (!is_string($value)) {
            $value = (string)$value;
        }
```

It will be better, faster and more clarify for reading

Reply:
done